CRM/Core/Component.php

```diff
@@ -24,11 +24,6 @@ class CRM_Core_Component {
    */
   const COMPONENT_INFO_CLASS = 'Info';
 
-  /**
-   * @var array
-   */
-  public static $_contactSubTypes = NULL;
```

Comment:
Since this turns out to be not needed I just removed it - however I think the 

```
/**
 * Contact Subtypes.
 *
 * @var array|null
 */
```

syntax looks like it does what you wanted and complies with a popular static code analyser (https://github.com/kalessil/phpinspectionsea - which has over 1 million downloads within php storm). I  do base my style changes on external standards - mostly a combo of the full drupal standard and the various things that seem to be agreed best practices in phpstorm world (since those obviously reflect a largish community and are easily implemented)